truffle/src/main/java/org/jruby/truffle/nodes/core/ProcessNodes.java

```diff
@@ -84,6 +84,7 @@ protected Object clock_gettime_realtime(int clock_id, RubyBasicObject unit) {
             return timeToUnit(time, unit);
         }
 
+        @TruffleBoundary
```

Comment:
I added support in the ahead of time compilation system for the unsupported feature that made error handling code reachable. We still need the TruffleBoundary here at least for ahead of time compilation so that it doesn't create a deopt target for the native clock_gettime.